install_script_op_worker1.sh

```diff
@@ -505,6 +505,8 @@ if [ ! -e "$pipeline_file" ]; then
   fi
 fi
 
+$sudo_cmd chmod 640 "$env_file"
```

Comment:
Not sure if I'm paranoid or not, should you `chmod` before appending any secret to the file?

Reply:
Agreed. Let's first create the file, then chmod and only then start writing values into it.

Reply:
Moved in https://github.com/DataDog/agent-linux-install-script/pull/72/commits/5b520df72356b3dddcd4219a0cef7cbb262c7fec